test/test_jit.py

```diff
@@ -1291,7 +1325,9 @@ def forward(self, x):
         # to insert quant-dequant nodes for quantizable tensors. The type analysis
         # happens as part of this jit pass
         torch._C._jit_pass_constant_propagation(trace.graph)
-        self.run_pass('insert_quantdequant', trace.graph)
+
+        qparam_dict = _helper_generate_qparam(trace, x1)
```

Comment:
Instead of actually running the model with observers, could you just create the appropriate IR with a qparam_dict as-if it was produced by observer-pass? The test should ideally test **only** the pass in question, not all the stuff it might work with.